mw-webapp/src/styles/_variables.scss

```diff
@@ -24,6 +20,7 @@ $tagBorderRadius: 1000px;
 
 // Stroke, border
 $primaryBorder: 1px;
+$secondaryBorder: 2px;
```

Comment:
Where do we need it? Designers told that everywhere should be stroke should be 1px

Reply:
fixed